MIGRATION.md

```diff
@@ -2,12 +2,11 @@
 
 ## Why?
 
-Newman v4.0 has major breaking changes which will affect you if you are using deprecated CLI options or running Newman on Node v4.
-It drops support for Node v4 and deprecated v2 CLI options. Also, inbuilt HTML reporter is moved to a standalone reporter.
+Newman v4.0 drops support for Node v4 and deprecated v2 CLI options. Also, inbuilt HTML reporter is moved to a standalone reporter.
```

Comment:
`the inbuilt HTML reporter has been moved`

Reply:
Also, mention the minimum supported Node version with a link to the compatibility matrix.